tests/unit_tests/test_db_handler.py

```diff
@@ -0,0 +1,276 @@
+
+
+def test_update_parameter_db(db, DB_CTA_SIMULATION_MODEL):
+
+    logger.info("----Testing updating a parameter-----")
+    db.copyTelescope(
+        DB_CTA_SIMULATION_MODEL,
+        "North-LST-1",
+        "Current",
+        "North-LST-Test",
+        "sandbox",
+    )
+    db.addParameter(
+        "sandbox", "North-LST-Test", "camera_config_version", "test", 42
+    )
+    db.updateParameter(
+        "sandbox", "North-LST-Test", "test", "camera_config_version", 999
+    )
+    pars = db.readMongoDB(
+        "sandbox", "North-LST-Test", "test", io.getTestOutputDirectory(), False
+    )
+    assert pars["camera_config_version"]["Value"] == 999
+
+    # TODO - what is tested here? or is this cleanup?
+    query = {"Telescope": "North-LST-Test"}
+    db.deleteQuery("sandbox", "telescopes", query)
+
+
+def test_adding_new_parameter_db(db, DB_CTA_SIMULATION_MODEL):
+
+    logger.info("----Testing adding a new parameter-----")
+    db.copyTelescope(
+        DB_CTA_SIMULATION_MODEL,
+        "North-LST-1",
+        "Current",
+        "North-LST-Test",
+        "sandbox",
+    )
+    db.addNewParameter(
+        "sandbox", "North-LST-Test", "test", "camera_config_version_test", 999
+    )
+    pars = db.readMongoDB(
+        "sandbox", "North-LST-Test", "test", io.getTestOutputDirectory(), False
+    )
+    assert pars["camera_config_version_test"]["Value"] == 999
+
+    query = {"Telescope": "North-LST-Test"}
+    db.deleteQuery("sandbox", "telescopes", query)
+
+
+def test_update_parameter_field_db(db, DB_CTA_SIMULATION_MODEL):
+
+    logger.info("----Testing modifying a field of a parameter-----")
+    db.copyTelescope(
+        DB_CTA_SIMULATION_MODEL,
+        "North-LST-1",
+        "Current",
+        "North-LST-Test",
+        "sandbox",
+    )
+    db.copyDocuments(
+        DB_CTA_SIMULATION_MODEL,
+        "metadata",
+        {"Entry": "Simulation-Model-Tags"},
+        "sandbox",
+    )
+
+    db.updateParameterField(
+        dbName="sandbox",
+        telescope="North-LST-Test",
+        version="Current",
+        parameter="camera_pixels",
+        field="Applicable",
+        newValue=False
+    )
+    pars = db.readMongoDB(
+        "sandbox", "North-LST-Test", "Current", io.getTestOutputDirectory(), False
+    )
+    assert pars["camera_pixels"]["Applicable"] is False
+
+    query = {"Telescope": "North-LST-Test"}
+    db.deleteQuery("sandbox", "telescopes", query)
+    query = {"Entry": "Simulation-Model-Tags"}
+    db.deleteQuery("sandbox", "metadata", query)
+
+
+def test_reading_db_sites(db):
+
+    logger.info("----Testing reading La Palma parameters-----")
+    pars = db.getSiteParameters("North", "Current")
+    if cfg.get("useMongoDB"):
+        assert pars["altitude"]["Value"] == 2158
+    else:
+        assert pars["altitude"] == 2158
+
+    logger.info("Listing files written in {}".format(io.getTestOutputDirectory()))
+    # subprocess.call(["ls -lh {}".format(io.getTestOutputDirectory())], shell=True)
```

Comment:
Removing these calls is not a bad idea (they produce a lot of noise). However, I guess they should be replaced by something. Also, in another test above we did have such lists, so we should be consistent.

Reply:
I plan to look at it as part of #292. I wrote it here as well cause I wanted to hear your opinion whether we should list the output or not like you did elsewhere. It's not critical though of course.

Reply:
OK forget this comment, I just saw that you removed the printout so I will do that as well.